### ISSUE TITLE
About the data volume.

### ISSUE THREAD

**Issue by tyvanzou** [abukharin3/asteroid#1]
# Questions about data volume and Performance

Hi,
Really a very novel, reliable, detailed and effective work! I think the results have fully prove the idea. I plan to use the pre-training architecture. But I'm wondering how many labels of DFT or Empirical data are needed for pre-training. In addition, I note the performance of tables and figures in the paper is a little different.

## Data Volume
In the paper **5.1 Datasets and Models**. I note that 
<img width="559" alt="image" src="https://github.com/user-attachments/assets/4b268862-82c2-4ab8-ad80-994b2de257ea">
As an example, 1000 DFT data points * (1/20) should equals 50 data points of CCSD data points, instead of 200 data points said in the paper. May I asked what do you mean about "The computational budget of standard training and ASTEROID are therefore equivalent."?
By the way, since not like the DFT result, which has using different data volume for pretraining, I am wondering does 100,000 configurations of empirical data points enough for pretraining? Should I use less or more data points of empirical force field to pretrain?

## Performance
In the paper, Table 1 (Test MAE of ASTEROID with empirical force field data.) and Table 2(Accuracy of ASTEROID with unlabeled molecular configurations.) have shown the performance of standard Tr. But in Figure 3 (Main results for GemNet when DFT data is viewed as inaccurate.) and Table 5(Accuracy of ASTEROID when the inaccurate data is comprised of multiple molecules), the performance of Standard Training is a little different. Why?

Thanks for your replying.
